filters/src/main/java/org/ricardogarfe/java/ReduceChapter.java

```diff
@@ -11,6 +11,7 @@ public Integer reduceListInitialValue(List<Integer> numbers, BinaryOperator<Inte
   }
 
   public Optional<Integer> reduceListWithoutInitialValue(List<Integer> numbers, BinaryOperator<Integer> binaryOperator) {
+    System.out.println("reduceListWithoutInitialValue");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/ricardogarfe-github/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.ricardogarfe.java%3Ajava-8-in-action&pullRequest=7&issues=AWOTkZKymXhJxGlVRijq&open=AWOTkZKymXhJxGlVRijq)

[ISSUE_KEY]: AWOTkZKymXhJxGlVRijq